src/ng/q.js

```diff
@@ -381,7 +381,11 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
       if (!toCheck.pur) {
         toCheck.pur = true;
         var errorMessage = 'Possibly unhandled rejection: ' + toDebugString(toCheck.value);
-        exceptionHandler(errorMessage);
+        if (toCheck.value instanceof Error) {
```

Comment:
I doubt it's common but if it turns out to be an issue we can revisit.

Ideally we would convert to a string (via `toDebugString`) only certain types of values and log others unchanged. The problem is that then we'd have to pass `Possibly unhandled rejection` and `toCheck.value` as separate arguments to `$exceptionHandler` and if we do it in a documented way (error first, (optional) reason second) the message would get swapped from e.g.:
```
'Possibly unhandled rejection: MY_THING'
```
to:
```
'MY_THING' 'Possibly unhandled rejection'
```
which is less pleasant.

We can discuss it but let's do it in a separate issue if desired.